keyboards/redragon/k552/v2/rules.mk

```diff
@@ -0,0 +1,3 @@
+# Data-driven structure doesn't support EEPROM yet
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = sn32_flash
```

Comment:
`redragon/k552/v2` is not a valid target with this structure. Defining options here will confuse the build system. Please remove the `rules.mk`